tests/Api/Request/RequestBuilderTest.php

```diff
@@ -162,6 +170,41 @@ public function testBuild(
         $this->assertEquals($expectedParams, $result->getParams());
     }
 
+    public function getUrlProvider()
+    {
+        return [
+            'Webstore configuration $domainSsl variable is not an empty string' => [
+                'https://example.com',
+                '',
+                'https://service.findologic.com/ps/example.com/index.php'
+            ],
+            'Webstore configuration $domainSsl variable is an empty string and $domain variable is not an empty string' => [
+                '',
+                'http://example.com',
+                'https://service.findologic.com/ps/example.com/index.php'
+            ],
+            'Webstore configuration $domainSsl and $domain variables are empty strings' => [
+                '',
+                '',
+                'https://service.findologic.com/ps/xml_2.1/index.php'
+            ]
+        ];
+    }
+
+    /**
+     * @dataProvider getUrlProvider
+     */
+    public function testGetUrl($domainSsl, $domain, $generatedUrl)
+    {
+        $this->webstoreConfigurationService->expects($this->once())->method('getWebstoreConfig')->willReturn(
+            new WebstoreConfigMock($domainSsl, $domain)
+        );
+
+        $requestBuilderMock = $this->getRequestBuilderMock();
+
+        $this->assertEquals($requestBuilderMock->getUrl(), $generatedUrl);
```

Comment:
The order of the parameters is wrong. The expected value should be the first one.